.gitpod.yml

```diff
@@ -0,0 +1,27 @@
+tasks:
+  - init: |
+      mvn -am -pl war,bom -P quick-build clean install
+    command: |      
+      mvn -pl war jetty:run -Dhost=0.0.0.0
+    name: Run
+  - command: gp await-port 8080 && gp url 8080 && gp preview $(gp url 8080)/jenkins/
+    name: Preview
+
+github:
+  prebuilds:
+    pullRequestsFromForks: true
+    addBadge: true
+
+jetbrains:
+  intellij:
+    plugins:
+      - Stapler plugin for IntelliJ IDEA
+    prebuilds:
+      version: stable
+
+vscode:
+  extensions:
+    - vscjava.vscode-java-pack
```

Comment:
![prompt](https://user-images.githubusercontent.com/154109/173089841-218b164e-fa82-4e83-a818-45c7ba2e791c.png)
despite this

Reply:
not for me, but I assume resolved given follow up comments?

Reply:
> I assume resolved

No, I just clicked **Install** and proceeded.